core/bubbles/bubble.ts

```diff
@@ -0,0 +1,591 @@
+/**
+ * @license
+ * Copyright 2023 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import {BlockDragSurfaceSvg} from '../block_drag_surface.js';
+import * as browserEvents from '../browser_events.js';
+import {IBubble} from '../interfaces/i_bubble.js';
+import {ContainerRegion} from '../metrics_manager.js';
+import {Scrollbar} from '../scrollbar.js';
+import {Coordinate} from '../utils/coordinate.js';
+import * as dom from '../utils/dom.js';
+import * as math from '../utils/math.js';
+import {Rect} from '../utils/rect.js';
+import {Size} from '../utils/size.js';
+import {Svg} from '../utils/svg.js';
+import * as userAgent from '../utils/useragent.js';
```

Comment:
pls fix lint

Reply:
I'm going to change this to an error so that it fails CI.